queries/subsquid/general/collectionById.graphql

```diff
@@ -14,6 +14,7 @@ query collectionById(
     ...collection
     ...collectionDetails
     createdAt
+    max
```

Comment:
seems like `max` currently still not available in the other queries. for example, error on this page: https://deploy-preview-6137--koda-canary.netlify.app/bsx/collection/2551182625?collectionId=2551182625

make another specific query chain?

Reply:
Yup max is available for stmn
and that is the purpose of this web